### PR TITLE
Update WeatherForecastController.cs

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Review by danroth27 — APPROVED** [review]

**Comment by pranavkm** [comment]
FYI @mkArtakMSFT. Merging this in since the blazor-hosted templates are missing a `using` which causes it to fail a build

File: src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -16,11 +17,11 @@ public class WeatherForecastController : ControllerBase
             "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
         };
 
-        private readonly ILogger<WeatherForecastController> _logger;
+        private readonly ILogger<WeatherForecastController> logger;
 
         public WeatherForecastController(ILogger<WeatherForecastController> logger)
         {
-            _logger = logger;
+            this.logger = logger;
```
Comment (davidfowl) [inline src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs]: woah woah woah

Comment (pranavkm) [reply src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs]: It's the right thing to do!

**Review by mkArtakMSFT — APPROVED** [review]

**Comment by mkArtakMSFT** [comment]
Feel free to merge this as soon as the CI checks pass.

File: src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (Eilon) [reply src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs]: Since when?!

**Pull request merged**